content/docs/start/ml-pipelines.md

```diff
@@ -0,0 +1,247 @@
+---
+title: 'Get Started: ML Pipelines'
+---
+
+# Get Started: ML Pipelines
```

Comment:
let's name it explicitly - `Metrics, parameters, plots` ? Just to be explicit. ML Pipeline is a bit ambiguous and I'm afraid can overlap with the pipelines we already have. 

especially considering that now, metrics for example don't require pipelines.

Reply:
One issue with this title is the sidebar shows as `Metrics Parameters Plots`.

Reply:
hmm, looks like a bug, mind creating a ticket in this repo for this? we'll take a look

Reply:
It's because I'm using the slug `metrics-parameters-plots`. Should I use a different slug? I can use `metrics,-parameters,-and-plots`, but I don't like the idea of using commas in the slug.

Or are you saying that the sidebar title shouldn't be based on the slug?

Reply:
To be honest I don't remember the exact logic how title is made, but looks like you are right - it's taken from a slug. Sidebar.json support custom labels - check the file, you will see some examples.

Reply:
Ah, can't believe I missed that!